stdlib/LinearAlgebra/src/dense.jl

```diff
@@ -291,6 +291,10 @@ function diagm_container(kv::Pair{<:Integer,<:BitVector}...)
     return falses(n, n)
 end
 
+# Defining diagm() for an AbstractVector
+
+diagm(v::AbstractVector) = diagm(0=>v)
```

Comment:
This got unfixed.

Reply:
How the hell?

Reply:
I force-pushed the latest commit!

Reply:
Any alternative?

Reply:
You made a remote commit through the GitHub UI. The force push overwrote it. You'd want to pull the change and do an interactive rebase to drop the unwanted commit.

Reply:
Ok!